setup.cfg

```diff
@@ -1,3 +1,22 @@
+[metadata]
+name = hai
+version = attr: hai.__version__
+author = Valohai
+author_email = dev@valohai.com
+maintainer = Aarni Koskela
+maintainer_email = akx@iki.fi
+description = Toolbelt library
+url = https://github.com/valohai/hai
+
+[options]
+packages = find:
+python_requires = >=3.6
+include_package_data = true
+
+[options.packages.find]
+where = .
+exclude = hai_tests
```

Comment:
I take it that this exclude will auto-include child items?

Reply:
Aye:

```
Archive:  dist/hai-0.2.0-py3-none-any.whl
Zip file size: 12943 bytes, number of entries: 12
-rw-r--r--  2.0 unx       22 b- defN 21-Nov-25 13:42 hai/__init__.py
-rw-r--r--  2.0 unx     6834 b- defN 21-Nov-25 10:00 hai/boto3_multipart_upload.py
-rw-r--r--  2.0 unx     1346 b- defN 21-Nov-25 10:00 hai/event_emitter.py
-rw-r--r--  2.0 unx     8300 b- defN 21-Nov-26 09:14 hai/parallel.py
-rw-r--r--  2.0 unx     9667 b- defN 21-Nov-26 09:14 hai/pipe_pump.py
-rw-r--r--  2.0 unx        0 b- defN 21-Nov-25 09:59 hai/py.typed
-rw-r--r--  2.0 unx     7027 b- defN 21-Nov-25 10:00 hai/rate_limiter.py
-rw-r--r--  2.0 unx     1075 b- defN 21-Nov-26 10:15 hai-0.2.0.dist-info/LICENSE
-rw-r--r--  2.0 unx      306 b- defN 21-Nov-26 10:15 hai-0.2.0.dist-info/METADATA
-rw-r--r--  2.0 unx       92 b- defN 21-Nov-26 10:15 hai-0.2.0.dist-info/WHEEL
-rw-r--r--  2.0 unx        4 b- defN 21-Nov-26 10:15 hai-0.2.0.dist-info/top_level.txt
?rw-rw-r--  2.0 unx      884 b- defN 21-Nov-26 10:15 hai-0.2.0.dist-info/RECORD
12 files, 35557 bytes uncompressed, 11479 bytes compressed:  67.7%
```